src/libs/OptionsListUtils.js

```diff
@@ -18,8 +18,45 @@ import Permissions from './Permissions';
  */
 
 let currentUserLogin;
+Onyx.connect({
```

Comment:
Why did we need to move these up? It was a linter thing?

Reply:
I think this is the right place for it. It seemed off so I asked about it and it was concluded that these should be up here:
https://expensify.slack.com/archives/C01GTK53T8Q/p1623436916221700